### ISSUE TITLE
Enhancement : Add option to save item to storage

### ISSUE THREAD

**Issue by HimanshuC22** [VictorImm/grocery-list#5]
There should be an option to add items to storage so that the user doesn't have to type every time for adding items with frequent use to the grocery list.
@VictorImm I would like to try this. Let me know if any clarification is required.

**Comment by VictorImm** [comment]
So you would add the dropdown menu then?
And a new page to configure the list of saved items?

**Comment by HimanshuC22** [comment]
Yes, I can add a dropdown menu, or in the item addition fragment I can add a button to add to list with save to storage. I prefer dropdown menu for now.
And yes, a new page to configure them, also.

**Comment by VictorImm** [comment]
Okay then
Happy coding!

**Comment by HimanshuC22** [comment]
@VictorImm I have added the pull request #9 with the screenshots of the added features/changes. Let me know if there is any problem of clarification needed.
